crates/sui-types/src/intent.rs

```diff
@@ -64,6 +65,9 @@ pub enum IntentScope {
     TransactionEffects = 1,
     CheckpointSummary = 2,
     PersonalMessage = 3,
+    SenderSignedTransaction = 4,
+    ProofOfPossession = 5,
+    Other = 6,
```

Comment:
I wouldn't put `Other` yet, it can open the can of worms if people use it. For each message type we should know its use, especially inside validator signature back and forth communication. Ideally we should tag every scope_use of a signature.

Reply:
removed, even though the scope for transaction effects digest and execution digest is not used, they are defined explicitly as it implements Message